README.md

```diff
@@ -1,2 +1,9 @@
-# bot
-Bot for the COVID19 Discord
+# SCICord
```

Comment:
@freememory Perhaps need to check how we want to refer to scicord? As SCICord or scicord. I always love lowercase.

Reply:
Resolved. Let's just go with `scicord`. Easy on everyone's eyes.